src/main/scala/sam/sceval/EvaluationPimps.scala

```diff
@@ -81,6 +84,9 @@ object EvaluationPimps extends Logging {
       binaryLabelCounts(cacheIntermediate, bins, recordsPerBin)
       .mapValues(blcs => blcs.map(BinaryConfusionMatrix(_, blcs.head)))
 
+
```

Comment:
@gm-spacagna  They are in the documentation, it does state it creates 4 stages.  I'll just delete the comments - I think they are left overs from when I was designing the algo.